src/ex_cmds.c

```diff
@@ -1575,45 +1596,61 @@ make_filter_cmd(
 	STRCAT(buf, itmp);
     }
 #else
-    // For shells that don't understand braces around commands, at least allow
-    // the use of commands in a pipe.
-    if (*p_sxe != NUL && *p_sxq == '(')
+    if (is_powershell)
```

Comment:
Okay, here is where I wanted to mention powershell being available for Linux, too (mistakenly did so on a previous line). Taking the `if is_powershell` block _above_ `#ifdef UNIX` should be enough. (Unless we don't care for people using PowerShell on Linux, assuming anyone does.)

Reply:
Moved the powershell handling code to outside of the '#ifdef/#endif'.